climada/hazard/centroids/centr.py

```diff
@@ -172,6 +177,40 @@ def from_base_grid(land=False, res_as=360, base_file=None):
             centroids = centroids.select(reg_id=land_reg_ids)
         return centroids
 
+    @staticmethod
```

Comment:
Why make it a @staticmethod?

Reply:
This follows the `from_base_grid` example. I also find that accessing such a special constructor as `cent = Centroids.from_xy()` is intuitive to read...

Reply:
I do not disagree. But, keeping the code homogeneous (i.e. not having different ways to initiate objects in CLIMADA for different types of objects) also has advantages, especially for not too python savvy people. Do you know if many other @Staticmethods except `from_base_grid` exist in CLIMADA?

Reply:
-> Topic for coding convention meetings 🙂